app/routes/session.py

```diff
@@ -126,7 +127,7 @@ def login() -> Response:
             "account_service_log_out_url"
         )
 
-    cookie_session["language_code"] = metadata.language_code
+    cookie_session["language_code"] = metadata.language_code or DEFAULT_LANGUAGE_CODE
```

Comment:
The only problem with this is that it doesn't fix it for people unless they relaunch. I think it is fine to leave as it and accept that. Not worth defaulting in get locale also.